ivy/functional/backends/numpy/manipulation.py

```diff
@@ -308,3 +308,9 @@ def as_strided(
         shape=shape,
         strides=strides,
     )
+
+
+def block(
+    arrays: Union[List[np.ndarray]], 
+) -> np.ndarray:
+    return np.block(arrays)
```

Comment:
This is a `numpy` backend implementation for `block`. I don't think we have added an `ivy.block` yet, so this would not work for the `numpy` frontend implementation. If you think we should add `block` to the Ivy Functional API, please consider looking at the [Ivy Experimental API Guide](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#ivy-experimental-api).